R/resp-stream-aws.R

```diff
@@ -57,15 +67,15 @@ parse_aws_event <- function(bytes) {
 
   # headers
   headers <- list()
-  while(i <= 12 + header_length) {
+  while (i <= 12 + header_length) {
```

Comment:
Sorry about the spurious style diffs; I've been experimenting with format on save.